modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee

```diff
@@ -956,7 +956,9 @@ class FileRowSummaryController extends Backbone.View
 			currentFileName: currentFileName
 			fileLink: window.conf.datafiles.downloadurl.prefix + "cmpdreg_bulkload/" + fileName
```

Comment:
Is `fileLink` in here twice? Looks like it

Reply:
@bffrost Good catch, thanks! Just fixed it.